docs/source/pillars/computing_technology.md

```diff
@@ -107,6 +107,14 @@ _The ability to run analyses requiring more compute resources, or more specialis
     For example, when using physical compute resources all sensitive data must be securely wiped before another user is given access to that same node.
     In a cloud hosted TRE virtual machines should be destroyed and recreated.
   - Mandatory
+* - A TRE could integrate with large-scale data analytics tools.
+  - For example Spark and Hadoop can be used for distributed computing across a cluster.
+    Where data included in TREs are impractically large for single-machine computing, this will be an advantage.
+  - Optional
+* - A TRE could integrate with cloud-native managed services.
```

Comment:
What does this really mean as part of the specification? Some TREs *can* do this and some can't. Some might want to and some might not. Is there any point in saying "do this if you want to"? There might be a point here about "only use cloud services if you're confident that you understand the security risks" but otherwise I'm not sure this is a useful part of the specification.

Reply:
I think our optional components should really be saying "we think you should X but we understand that you might not want to". I'm not sure that's true as written, but there *is* something around "consider doing X but if-and-only-if you understand the risks".